test/calibration/experiments/test_drag.py

```diff
@@ -73,6 +74,7 @@ def test_end_to_end(self):
         drag = RoughDrag(0, self.x_plus)
         drag.analysis.set_options(p0={"beta": 1.2})
         exp_data = drag.run(backend)
+        self.assertSuccess(exp_data)
```

Comment:
The other option to fix the failures would be to manually block for results before calling `assertSuccess` as:
```python
exp_data = drag.run(backend)
exp_data.block_for_results()
self.assertSuccess(exp_data)
```